content/chronograf/v1.8/introduction/getting-started.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: Getting started with Chronograf
+title: Get started with Chronograf
+description: Get started with Chronograf.
 aliases:
     - /chronograf/latest/introduction/getting-started/
     - /chronograf/v1.8/introduction/getting_started/
```

Comment:
I know you didn't add these, but these aliases need to be removed. They will cause a redirect loop (the page redirecting to itself).